rtsp_simulator_ffmpeg_v2.py

```
@@ -31,6 +31,8 @@
 import subprocess
 import shutil
 import psutil
+import dotenv
```

Ruff (0.12.2)

`dotenv` imported but unused

Remove unused import: `dotenv`

(F401)